extension.js

```diff
@@ -151,49 +157,55 @@ var ContainerSubMenuMenuItem = GObject.registerClass(
+                stopBtn.reactive = false;
+                pauseBtn.reactive = false;
+                this.insert_child_at_index(createIcon("media-playback-stop-symbolic", "status-stopped"), 1);
                 break;
             }
             case "Up":
             case "running": {
-                this.menu.addMenuItem(new PopupMenuItem("Started", container.startedAt));
+                startBtn.reactive = false;
+                deleteBtn.reactive = false;
+                pauseBtn.checked = false;
                 this.insert_child_at_index(createIcon("media-playback-start-symbolic", "status-running"), 1);
-                const pauseMenuIten = new ContainerMenuItem(container.name, "pause", () => container.pause());
-                pauseMenuIten.insert_child_at_index(createIcon("media-playback-pause-symbolic", "status-stopped"), 1);
-                this.menu.addMenuItem(pauseMenuIten);
-                const stopMenuItem = new ContainerMenuItem(container.name, "stop", () => container.stop());
-                stopMenuItem.insert_child_at_index(createIcon("process-stop-symbolic", "status-stopped"), 1);
-                this.menu.addMenuItem(stopMenuItem);
-                const restartMenuItem = new ContainerMenuItem(container.name, "restart", () => container.restart());
-                restartMenuItem.insert_child_at_index(createIcon("system-reboot-symbolic", "status-restart"), 1);
-                this.menu.addMenuItem(restartMenuItem);
-                this.menu.addMenuItem(createTopMenuItem(container));
-                this.menu.addMenuItem(createShellMenuItem(container));
-                this.menu.addMenuItem(createStatsMenuItem(container));
                 break;
             }
             case "Paused":
             case "paused": {
                 this.insert_child_at_index(createIcon("media-playback-pause-symbolic", "status-paused"), 1);
-                const unpauseMenuItem = new ContainerMenuItem(container.name, "unpause", () => container.unpause());
-                unpauseMenuItem.insert_child_at_index(createIcon("media-playback-start-symbolic", "status-start"), 1);
-                this.menu.addMenuItem(unpauseMenuItem);
                 break;
             }
             default:
                 this.insert_child_at_index(createIcon("action-unavailable-symbolic", "status-undefined"), 1);
                 break;
             }
 
-            // add log button
-            const logMenuItem = createLogMenuItem(container);
-            this.menu.addMenuItem(logMenuItem);
+            details.actor.add_child(new ContainerMenuItem(container.name, "Logs", () => container.logs()));
+            details.actor.add_child(new ContainerMenuItem(container.name, "Top", () => container.watchTop()));
+            details.actor.add_child(new ContainerMenuItem(container.name, "Shell", () => container.shell()));
+            details.actor.add_child(new ContainerMenuItem(container.name, "Statistics", () => container.stats()));
```

Comment:
Menu items should contain a verb, for example "Show logs", "Open shell", "Show statistics", "Watch top" ("Show processes"?)

Reply:
done